liota/dccs/iotcc.py

```diff
@@ -83,15 +83,16 @@ def on_response(msg):
                     return True
                 else:
                     log.debug("Processed msg: {0}".format(json_msg["type"]))
-                    on_response(self.recv_msg_queue.get())
+                    on_response(self.recv_msg_queue.get(True,10))
```

Comment:
How can we receive create_relationship_Response at time of establishment of a new connection? Is it during re-connection? Are you receiving it for both MQTT and Websocket? I believe it might be only for MQTT, as in MQTT we are re-using the topic which was used during previous connection. For Websocket, due to a different transaction id, a re-connection will not receive messages from previous connection.